tests/CacheCompat/CommonCache.Test.AdalV3/FileBasedAdalV3TokenCache.cs

```diff
@@ -25,16 +25,18 @@
 // 
 // ------------------------------------------------------------------------------
 
+using System;
 using System.IO;
 using System.Security.Cryptography;
+using CommonCache.Test.Common;
 using Microsoft.IdentityModel.Clients.ActiveDirectory;
 
 namespace CommonCache.Test.AdalV3
 {
     // This is a simple persistent cache implementation for an ADAL V3 desktop application
     public class FileBasedAdalV3TokenCache : TokenCache
     {
-        private static readonly object FileLock = new object();
+        private static readonly object s_fileLock = new object();
```

Comment:
Does it still make sense if several apps can share the cache?

Reply:
Not sure I understand the question.  We still need to lock the file to ensure only one person is reading/writing.